src/sdk/pynni/nni/nas/pytorch/classic_nas/mutator.py

```diff
@@ -37,70 +44,102 @@ def __init__(self, model):
         use ```nnictl``` to start an experiment. The other is standalone mode
         where users directly run the trial command, this mode chooses the first
         one(s) for each LayerChoice and InputChoice.
-
         Parameters
         ----------
-        model : pytorch model
+        model : PyTorch model
             user's model with search space (e.g., LayerChoice, InputChoice) embedded in it
         """
         super(ClassicMutator, self).__init__(model)
-        self.chosen_arch = {}
-        self.search_space = self._generate_search_space()
-        if 'NNI_GEN_SEARCH_SPACE' in os.environ:
+        self._chosen_arch = {}
+        self._search_space = self._generate_search_space()
+        if "NNI_GEN_SEARCH_SPACE" in os.environ:
```

Comment:
change the "NNI_GEN_SEARCH_SPACE" as an enum string?

Reply:
updated